spec/rubocop/cop/layout/closing_parenthesis_indentation_spec.rb

```diff
@@ -67,6 +67,13 @@
         expect_no_offenses('some_method()')
       end
 
+      it 'accepts empty () with newlines' do
+        expect_no_offenses(<<-RUBY.strip_indent)
+          some_method(
+          )
+        RUBY
+      end
```

Comment:
Thank you for fixing the error 👍 

It seems better to pass the tests on the following cases as well.

```ruby
# bad
foo = some_method(
)

# good
some_method(
           )

foo = some_method(
      )

foo = some_method(
                 )
```

Reply:
@koic

Thank you for your reviews!
I have a question about the following bad case you mentioned.

```
# bad
foo = some_method(
)
```

I thought that the code was a good case, and added a test as a good case: [here](https://github.com/tatsuyafw/rubocop/blob/813170b5c247d4ce7f873ae28a81aceda0a93528/spec/rubocop/cop/layout/closing_parenthesis_indentation_spec.rb#L212-L217)

Is it better that tests pass the cases you wrote?

Reply:
Oops, this is my mistake.

I thought of the following comparison. It was a bad case to align the variable and the indentation of `(`. So I thought was bad.

## bad

```console
% cat bad.rb
def foo
  foo = JSON.parse(a
  )
end

% rubocop bad.rb --only Layout/ClosingParenthesisIndentation
Inspecting 1 file
C

Offenses:

bad.rb:3:3: C: Layout/ClosingParenthesisIndentation: Align ) with (.
  )
  ^

1 file inspected, 1 offense detected
```

## good

```console

% cat good.rb
def foo
  foo = JSON.parse(a
                  )
end

% rubocop good.rb --only Layout/ClosingParenthesisIndentation
Inspecting 1 file
.

1 file inspected, no offenses detected
```

However, considering the following case, it is good to align the variable and the indentation of `(`, which is not a problem.

## good

```
% cat good2.rb
def foo
  foo = JSON.parse(
    a
  )
end

% rubocop good2.rb --only Layout/ClosingParenthesisIndentation
Inspecting 1 file
.

1 file inspected, no offenses detected
```

Therefore, the following case is good.

```ruby
foo = some_method(
)
```

Reply:
Thank you for your kind explanations. I understand. I try to fix codes to pass the following tests.

```
# bad
some_method(
  )

foo = some_method(
  )

# good
some_method(
)

some_method(
           )

foo = some_method(
)

foo = some_method(
      )

foo = some_method(
                 )
```

Reply:
This looks great. Thank you for explaining with an easy to understand examples! Please go ahead with it.

Reply:
@koic I'm sorry for my late reply :bow:

I've fixed codes to pass the tests described in the above. I kindly ask your confirmation.

Reply:
Don't worry. Thank you very much for your great work! I have written a review comment.
https://github.com/rubocop-hq/rubocop/pull/6130/files#r207432492